src/lusty/filesystem-explorer.rb

```diff
@@ -184,6 +188,31 @@ def all_files_at_view
       end
     end
 
+    def all_remote_files(view, view_str)
+      host, path = view_str[6..-1].split('/', 2)
+      option = if LustyM::option_set?("AlwaysShowDotFiles") or \
+         current_abbreviation()[0] == ?.
+                'a'
+               else
+                 ''
+               end
+
+      files_s = `ssh #{host} ls -1FL#{option} -- #{path}`
```

Comment:
`-F` is used to tell if a file is a directory or not. Maybe we can drop other symbols, or we should instead add them to local ones, since users don't usually view a pipe or socket in vim.

`-L` is to follow symlinks, so a symlink to a directory shows up as a directory, consistent with the local behavior.